docs/guides/modules/test/pages/fix-flaky-tests.adoc

```diff
@@ -0,0 +1,238 @@
+** Make sure your organization link:https://help.openai.com/en/articles/10910291-api-organization-verification#h_c6efff0719[has gpt-5 model access].
+** Verify your organization. For guidance see the link:https://help.openai.com/en/articles/10910291-api-organization-verification[OpenAI help]. If you cannot get your OpenAI organization verified, see the troubleshooting item <<i-cannot-get-my-openai-organization-verified>>.
+* Ensure your CircleCI jobs store test results using the `store_test_results` step. Read more about this step in the xref:reference:ROOT:configuration-reference.adoc#storetestresults[configuration reference].
+* Ensure you have the CircleCI GitHub app installed in your GitHub organization. Check menu:Organization Settings[VCS Connections], where you can see if you have the App already installed, or select btn:[Install GitHub App]. The CircleCI agent needs the GitHub App to be installed to be able to recommend fixes and open pull requests.
+* Make sure you are following projects in your organization that do have flaky tests to fix. CircleCI identifies flaky tests in your CI/CD pipelines on the *Tests* tab for workflows in the Insights dashboard (menu:Insights[Select project > Select workflow > Tests]).
+
+=== Setup
+
+. In the link:https://app.circleci.com/home[CircleCI web app], choose your organization and select *Organization Settings* from the sidebar.
+. Select *Chunk Tasks* from the sidebar.
+. Select btn:[Get started] and then btn:[Continue] when prompted.
+. If you followed the prerequisites, you should see a image:guides:ROOT:icons/passed.svg[passed icon, role="no-border"] to indicate you already have the GitHub App installed for your organization. If not use the btn:[Install CircleCI GitHub App] button to install it.
+. Select your AI Model provider (Anthropic or OpenAI).
+. Enter your API key for your chosen model provider.
+. Select btn:[Next] to complete the setup.
+
+#Screenshots#
+
+=== Assign a task
+
+Once you have Chunk installed and set up for your organization you can start assigning tasks.
+
+To set up a "Fix flaky tests" task follow these steps:
+
+. Select the project you want to assign the task to.
+. Choose a run frequency (daily, weekly, monthly).
+. Choose the number of tests you want Chunk to try to fic per run, between one and three.
+. Choose a maximum for the number of solutions you want Chunk to try for each test, between one and three.
+. Choose a number of validation runs to allow per test between one and 20.
+. Choose a maximum number of concurrent open PRs for flakey test fixes. You can choose between one and 20, or the default, "Unlimited".
+. In the "Custom settings" section you can add a setup script for the Chunk agent to run before each run. Youc an also add post-run commands to run after each run.
```

Comment:
What's the relationship between this and the guidance we now have to add a cci-agent-setup.yml to the repo?